src/main/resources/resource/framework/pom.xml.template

```diff
@@ -420,7 +427,8 @@
                   </filesets>
                 </configuration>
       </plugin>
-
+      <!-- attached to Maven test phase -->
+      <!--

```

Comment:
jacoco commented out from here?  is jenkins handling this for us instead?

Reply:
was having a hard time with jacoco in jenkins and gitactions ... need to take it out for a bit